src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -2707,6 +2765,7 @@ ps_ert_submit(struct sched_cmd *cmd)
 	switch (opcode(cmd)) {
 	case ERT_CONFIGURE:
 		SCHED_DEBUG("<- %s (configure)\n", __func__);
+		kill_soft_kernel(cmd);
```

Comment:
This is OK. But I think a better place would be in config function itself (around line 779). That is the place we handle reconfiguration in ert mode with comment.

Reply:
done